src/main/java/fr/minuskube/inv/util/Pattern.java

```diff
@@ -67,22 +143,93 @@ public T getObject(int row, int column) {
         return this.mapping.getOrDefault(this.lines[row].charAt(column), this.defaultValue);
     }
 
+    /**
+     * Searches through this patterns lines to find the first top-left occurrence of this key.
+     * If it could not be found, the retured {@link Optional} is empty.
```

Comment:
Small typo on "retured"

Reply:
Fixed it in the commit 3babb5369e5e4dec3f7cb1096ad507fe68f6c9c2